### PR TITLE
Fixed border on cells in events disappear

### PR DESCRIPTION
**Pull request by JuanGarriuz**
### Description

Clicking in a cell makes the border disappear, this PR overwrites EUI styles changing the default euiDataGridRowCell:focus
 
### Issues Resolved

- https://github.com/wazuh/wazuh-dashboard-plugins/issues/7049

### Evidence

image
image

### Test

- [ ] Check that clicking in a cell shouldn't make the border disappear

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by guidomodarelli — APPROVED** [review]
CR :green_circle: 
TEST :green_circle: 
image

**Comment by guidomodarelli** [comment]
image

**Comment by JuanGarriuz** [comment]
Resolved

image

File: plugins/main/public/components/common/wazuh-discover/discover.scss [hunk 1]
```diff
@@ -3,6 +3,11 @@
     height: max-content !important;
   }
 
+  .euiDataGridRowCell:focus {
+    border: 1px solid rgba(0, 107, 180, 0.4);
```
Comment (Desvelao) [inline plugins/main/public/components/common/wazuh-discover/discover.scss:7]: **nitpick**: this defines a hardcoded color without managing the theme or light/dark mode.

**Comment by guidomodarelli** [comment]
image

Is not pixel perfect

**Comment by JuanGarriuz** [comment]
The left and bottom lines do not belong to the cell in question but to the adjacent ones. A box-shadow can be added, but there might be other styles that could be affected by this

image

image

**Review by asteriscos — APPROVED** [review]
Test: :green_circle: 

image

The focused style of the cell is the same as OpenSearch dashboard

**Review by guidomodarelli — APPROVED** [review]
Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

CR :green_circle:
TEST :green_circle:

File: plugins/main/public/components/common/wazuh-discover/discover.scss (hunk 1 above)
Comment (Desvelao) [reply plugins/main/public/components/common/wazuh-discover/discover.scss]: The style was removed in a later commit.

**Pull request merged**